error_tracker/django/migrations/0002_auto_20201018_1311.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.1.2 on 2020-10-18 11:11
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('django', '0001_initial'),
+    ]
+
+    operations = [
+        # migrations.AddField(
```

Comment:
We would need  to add migrations for exceptions table. And please remove alter for request_data field.